webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9718,6 +9717,41 @@ var userOverride = {
                 }
                 return '';
             };
+            _this.getMessageCreatedAtTime = function (createdAtTime) {
+                if (TIME_FORMAT_24_HOURS) {
+                    var msgtime = new Date(createdAtTime);
+                    var yesterday = new Date();
+                    if (yesterday.getDate() != msgtime.getDate()) {
+                        var time = msgtime.toLocaleString('en-US', {
+                            day: 'numeric',
+                            month: 'short'
+                        }) + ", " + msgtime.getHours() + ":" + msgtime.getMinutes();
+                        return time;
+                    }
+                    else if (yesterday.getDate() == msgtime.getDate() && yesterday.getMonth() != msgtime.getMonth()) {
```

Comment:
yesterday.getDate() == msgtime.getDate() is not required here, because this will be always true if it comes to this else if, otherwise it would have already been gone to the above if if (yesterday.getDate() != msgtime.getDate())

Reply:
yeah, right. will remove this.

Reply:
changed